playbooks/engineer-workflow.md

```diff
@@ -31,85 +31,51 @@ in 2 week sprints, with a planning meeting at the start and a review/retrospecti
 
 ## Workflow
 
-All developers have access to all repositories. Pull requests are always welcome. Check in with responsible teams
-or developers for help getting up to speed. Production repos should have a point-person or point-team in their
-README if you want to know who to start asking questions to.
+All developers have access to all repositories. Pull requests are always welcome. Check the docs as well as with
+responsible teams for help getting up to speed. Production repos should have a point-person or point-team in their
+README if you're unsure.
 
 ### Working in branches
 
-Artsy teams prefer to work in branches on a shared repository rather than forks. This means that developers should
-clone the repo, create a working branch from `main`, then push that branch back to the main repo.
-[Pull requests](#pull-requests) are then made from the working branch back to the `main` branch. For tidiness,
-working branches should be named by their creator like `<github_user_name>/<prefix>/<topic>`, even when a
-collaboration.
-
-Wait, what? What prefix? We use [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) prefixes in
-our PR titles, because by default, we squash our commits when merging a PR, and this means that the squashed commit
-automatically ends up using the Conventional Commit format. Using the prefix in your branch name is not essential,
-but it's, well, nice.
+Artsy teams prefer to work in branches on a shared repository rather than forks. Developers should clone the repo,
+create a working branch from `main`, then push that branch back to the shared repo. [Pull requests](#pull-requests)
+are then made from the working branch back to the `main` branch. For tidiness, working branches should be named by
+their creator like `<github_username>/<topic>`.
 
 ### Commit Messages
 
 > A well-crafted git commit message is the best way to communicate context about a change to fellow developers (and
 > indeed to [our] future selves). -[How to write a git commit message](https://chris.beams.io/posts/git-commit/)
 
-In support of this, Artsy engineering adopted
-[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) as its commit message convention
-(see RFC [Best Practices for Naming and Merging PRs](https://github.com/artsy/README/issues/327)). We encourage you
-to read the [spec](https://www.conventionalcommits.org/en/v1.0.0/#specification) for more detail but, in general,
-all commits should match the following format
-([see examples](https://www.conventionalcommits.org/en/v1.0.0/#examples)):
+Artsy prefers [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (see
```

Comment:
There's nothing anymoreafter "(see", right?

Reply:
There is--automatically wrapped to the next line. (I dislike this auto-formatting but 🤷 ).

Reply:
Shall we remove this auto-formating? I also don't like it. It makes it harder to read PRs, since it shows changes where there is none.